site/styles/_buttons.scss

```diff
@@ -244,3 +244,24 @@
     border-color: $gray-200;
   }
 }
+
+s72-donate-button:not(:empty) {
+  :hover {
+    background-color: $button-background-hover;
+    color: $button-text-color-hover;
+    svg {
+      g {
+        fill: $button-text-color-hover;
+      }
+    }
+  }
```

Comment:
to pass CI
/* stylelint-disable selector-max-compound-selectors */
...
/* stylelint-enable selector-max-compound-selectors */

Reply:
wrapped around the svg styling and looks like its passed 👍  chur